pkgs/misc/vscode-extensions/ms-dotnettools-csharp/update-bin-srcs-lib.sh

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env bash
```

Comment:
This should probably use nix-shell.

Reply:
nix-shell shebang added to `update-bin-srcs`. `update-bin-srcs-lib.sh` being a bash function lib sourced by `update-bin-srcs`.